### DISCUSSION TITLE
Issues Adding a New Page

### DISCUSSION THREAD

**Discussion by HendricoMaximus** [community/community#22186] [Other Feature Feedback, Questions, & Ideas]
<p>Hello there! Welcome to my first post here on the GitHub community! Currently, I have been working on revamping my website using GitHub Pages! I have run into some issues though. I cannot add a new page. I have the home and 404 In progress, but I cannot add another new page. According to <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/adding-content-to-your-github-pages-site-using-jekyll#adding-a-new-page-to-your-site" rel="noopener nofollow ugc">this page</a>, you add a new pull request, and that’s what I have done.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/5/8537075ea0c3ac609ff9773ea67cab5edcf9caeb.png" data-download-href="/uploads/short-url/j0toW4mDOEd46k5HDg0FCF6rEEP.png?dl=1" title="Screen Shot 2020-12-15 at 5.13.56 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75211263/181069406-c9b02679-ae22-42bb-87b5-47b823aece50.png" alt="Screen Shot 2020-12-15 at 5.13.56 PM" data-base62-sha1="j0toW4mDOEd46k5HDg0FCF6rEEP" width="690" height="370" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/8537075ea0c3ac609ff9773ea67cab5edcf9caeb_2_690x370.png, https://aws1.discourse-cdn.com/github/original/3X/8/5/8537075ea0c3ac609ff9773ea67cab5edcf9caeb.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/5/8537075ea0c3ac609ff9773ea67cab5edcf9caeb.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/8537075ea0c3ac609ff9773ea67cab5edcf9caeb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-12-15 at 5.13.56 PM</span><span class="informations">1016×546 53.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
As you can see in the above screenshot, I have added the branch, and I’ll list the basic code for the page below.
<pre><code class="lang-auto">layout: page
title: "Scripting Examples"
description: "Examples of my amazing scripting ability!"
permalink: /portfolio/scripting/
</code></pre>
<h4>Sorry for the Lua code block, it is the only block I know how to use on a discourse forum.</h4>
<p>As you can see in the code, I have added a title, description, and the <code>permalink</code>, but it still does not work. For those wanting to test, go to <a href="http://hendricomaximus.github.io/portfolio/scripting" rel="noopener nofollow ugc">http://hendricomaximus.github.io/portfolio/scripting</a>, which should be what the permalink makes.</p>
<p>Thanks for reading! Any help is appreciated! Have a nice day!</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/hendricomaximus">@HendricoMaximus</a>! Welcome to the community. You need to simply merge the pull request. Click on Pull Requests, then the one you opened, and you should be able to click the merge button.  Try the link you posted above in ~5-10 minutes and it will work.</p>

**Comment by HendricoMaximus** [comment]
<p>Sorry for the delay! Thank you for telling me! I didn’t see that in the tutorial.</p>

**Comment by logankilpatrick** [comment]
<p>No worries, happy to help!</p>

**Comment by logankilpatrick** [comment]
<p>For what it’s worth, you definitely did not make a mistake anywhere here. The tutorial you linked is incomplete and I opened an issue to resolve that here: <a href="https://github.com/github/docs/issues/2150" rel="noopener nofollow ugc">https://github.com/github/docs/issues/2150</a></p>
